src/data-loader.ts

```
@@ -516,9 +516,21 @@ export async function getEarliestTimestamp(filePath: string): Promise<Date | nul
 		return earliestDate;
 	}
 	catch (error) {
-		// Log file access errors for diagnostics, but continue processing
+		// Handle file access errors gracefully
 		// This ensures files without timestamps or with access issues are sorted to the end
-		logger.debug(`Failed to get earliest timestamp for ${filePath}:`, error);
+		
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)